.github/workflows/network-tests.yml

```diff
@@ -34,4 +38,8 @@ jobs:
     - name: checks
       run: |
         yarn install --frozen-lockfile --network-timeout 120000
+        yarn workspace @joystream/types build
+        yarn workspace @joystream/cd-schemas generate:all
+        yarn workspace @joystream/cd-schemas build
+        yarn workspace @joystream/cli build
```

Comment:
Is there a reason for building the CLI here?

Reply:
done in https://github.com/Joystream/joystream/pull/2302/commits/d9d6762b817b9d1e8c4867bbaa7379499dfcf9c6